src/approve-prs.ts

```diff
@@ -160,7 +160,7 @@ export async function main(cli: meow.Result) {
   const repos = await github.getRepositories();
   for (const repository of repos) {
     console.log(repository.name);
-    let prs: PullRequest[];
```

Comment:
Is there a reason we are dropping types?

Reply:
The TypeScript compiler now infers them from the assignment inside of the try!  It's actually a really nice feature.  The types returned from Octokit are a little weird.